### PR TITLE
Small change so we can circulate a PR

### PR DESCRIPTION
**Pull request by efixler**
# Goal

Get input and iterate on the next stage or PR guidelines and template.

I know that this PR isn't the ideal delta PR, since the first draft was committed prior. Please expand the file using the file viewer and leave comments as conversation. 

The easiest way to read what's being discussed here is probably to open the [root of the repo](https://github.com/MozillaSocial/pr-template), since the PR content is in the README.

## What's Here

Mainly what's here is a README describing the goals and a template file for PRs

## Steps to Test

Part of this review is shaking out the new GH org config, making sure people can see it and have adequate permissions and generally getting things working as we expand access to the whole team.

## What I'd Like Feedback On

Anything related to the guidelines and implementation thereof. Also, we'll definitely need a lot of help getting the GH org fully set up, stuff like:

- Automated checks e.g.
   - Lint
   - Docblock checks
   - Test coverage checks
   - etc.
- Per-language templates: Base gitignores, directory structure, etc. We will probably need these for
   - Typescript
   - Rust 
   - iOS
   - Android
 - Other GH Setup and config, like teams, rules, other actions, etc.

Help and input here would be amazing and appreciated, and will help us move fast and efficiently.
  

## Other References

None at this time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,11 +80,12 @@ PRs should include:
   > think?"
 
 - An approval means you're confident that this PR will both do what it's supposed to do, improve the Mozilla Social codebase, and not introduce technical debt.
-- Expect to start your review within one business day of the PR going to review and complete it within another. If you can't do that, please tell the author early on.
-  As an author, expect that meaningful review may take time. If there is so much
-  urgency that the code must be checked in as soon as possible this should be
-  both an exceptional situation and you should be collaborating with your
-  reviewer (and others) during the drafting process.
+- Ideally, a review should be completed within one business day of the request.
```
Comment (efixler) [inline README.md:83]: cc @ml8 @bassrock

Comment (bassrock) [reply README.md:83]: @efixler works for me

Comment (ml8) [reply README.md:83]: LGTM. (Is "by the end of the next business day" an acceptable compromise?)

**Review by ml8 — APPROVED** [review]
LGTM, but I had one question 👉👈

File: README.md (hunk 1 above)
Comment (efixler) [reply README.md:83]: @ml8 let's keep this as is for now, pending further input. I think my biggest hesitation around that suggestion isn't necessarily the raw time delta, it's the fact that we are all in different time zones, so "end of business day" can mean different things.

**Pull request merged**